pkg/scheduler/factory_test.go

```diff
@@ -354,18 +365,17 @@ func TestDefaultErrorFunc(t *testing.T) {
 		}
 	}
 
+	errFunc(testPodInfo, err)
 	// Try up to a minute to retrieve the error pod from priority queue
 	foundPodFlag := false
 	maxIterations := 10 * 60
 	for i := 0; i < maxIterations; i++ {
 		time.Sleep(100 * time.Millisecond)
-		got := getPodfromPriorityQueue(queue, testPod)
+		got := getPodFromPriorityQueue(queue, testPod)
 		if got == nil {
 			continue
 		}
 
-		testClientGetPodRequest(client, t, testPod.Namespace, testPod.Name)
```

Comment:
The `err := apierrors.NewNotFound(apicore.Resource("node"), nodeFoo.Name)` make me think that the intention was to test that RemoveNode was called?

Reply:
Not really. The `testClientGetPodRequest` function is just for "pod request". But using the same err for different test cases seems confusing, I will clean them up.

Reply:
It just seems that they took care of the coverage, but didn't test the effect.